dist/css/yeldaWebchatInjector.css

```diff
@@ -1,260 +1,5 @@
 
+/*# sourceMappingURL=yeldaWebchatInjector.css.map*/
```

Comment:
now it's ok, thanks BUT `css/yeldaWebchatInjector.css` should not be minified. It's the build, with the variable full names, and  `css/yeldaWebchatInjector.min.css` is it's minified version.
but well, that's not a big deal for now, so i'll accept the PR
thanks